go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/solo-io/solo-apis
+
+go 1.14
+
+replace (
+	github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309
+	k8s.io/client-go => k8s.io/client-go v0.17.1
```

Comment:
should we make this 0.18 to match `k8s.io/apimachinery`?

Reply:
Everything should be `0.17`. `0.18` has breaking changes to the client functions.

Reply:
I actually just fixed this locally. I am actually pinning `apimachinery` to `0.17.1` as well, as that is the version of the k8s libraries we use in skv2.